addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -101,10 +98,7 @@ private _id = switch (_type) do {
     case "turret": {
         if (_applyRetroactively) then {
             private _vehicle = vehicle GVAR(oldUnit);
-            private _turret = [];
-            if (GVAR(oldUnit) != _vehicle) then {
-                _turret = ([[-1]] + allTurrets [_vehicle, true]) select {_vehicle turretUnit _x == GVAR(oldUnit)} param [0, []];
-            };
+            private _turret = _vehicle unitTurret GVAR(oldUnit);
```

Comment:
```suggestion
            private _turret = _vehicle turretUnit GVAR(oldUnit);
```

Reply:
I get confused by these names (even though I guess it makes sense) but what is there now should be right
- unitTurret = turret path (array) [the new command]
- turretUnit = unit in turret (object)

Reply:
Ah right! :+1: I just thought your last commit broke them.